readme.md

```diff
@@ -684,6 +684,24 @@ if (colorSchema.check(value)) {
 }
 ```
 
+The enum type also accepts an options object as a second parameter.
+
+You can set `coerce` to `'lower'` or `'upper'` to ignore string casing when calling `check` or `parse`.
+
+```typescript
+z.enum(Colors, { coerce: 'lower' });
```

Comment:
The example in the readme uses `check` but coerce specifically does not work with the `check` api

Reply:
Ah thanks for catching this. Fix is up now.